internal/services/springcloud/spring_cloud_container_deployment_resource_test.go

```diff
@@ -204,25 +205,43 @@ resource "azurerm_spring_cloud_container_deployment" "test" {
 func (SpringCloudContainerDeploymentResource) template(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
```

Comment:
Normally this shouldn't be defined in the acctest, which usually indicates the API bug, that the service team shall fix?

Reply:
That's caused by the alert rule that's auto created by application insights, it's a known issue.

Reply:
Shall we put some comments about the reason together with the issue link?